File: PRAK403-2210817120013-Fathiah_Nuraisyah_Radam.py
angka1, angka2 = map(int, input().split())
if(angka1 < angka2):
    a = 1
    for i in range(angka1, angka2 + 1):
        x = angka2 - a + 1
        a +=  1
        if i == angka1:
            print(end="")
        else:
            print(" - ", end=" ")
        print(f"{i} {x}", end=" ")
else:
    a = 0
    for i in reversed(range(angka2, angka1 + 1)):
        x = angka2 + a
        a += 1
        if (i == angka1):
            print(end="")
        else:
            print(" - ", end=" ")
        print(f"{i} {x}", end=" ")

File: PRAK403-2210817120013-Fathiah_Nuraisyah_Radam.c
#include <stdio.h>
int main (){
      int angka1, angka2, i;
      int a = 0;
      scanf("%d %d", &angka1, &angka2);
      if (angka1 > angka2){
            for (i = angka1; i>= angka2; i--){
                  if (i == angka1){}
                  else {
                        printf(" - ");
                  }
                  printf("%d %d", i, angka2 + a); a++;
            }
      }
      else {
            for (i = angka1; i <= angka2; i++){
                  if (i == angka1){}
                  else {
                        printf(" - ");
                  }
                     printf("%d %d", i, angka1 + angka2 - i);
            }
      }
}